docs/Drawer.md

```diff
@@ -216,6 +196,34 @@ import DrawerFooter from '@pxblue/react-components/core/Drawer';
 </DrawerFooter>
 ```
 
+## Interitables
+There are a few optional props that controls the cosmetics of the drawer items and are shared between `Drawer`, `DrawerNavGroup`, `NavItem` and `NestedNavItem`. These properties are inheritable, meaning that the children can inherit the props from their parents if the child prop is set to undefined, and can override the parent props by setting child props to certain values. 
+
+| Name                      | Description                                                | Type               | Required | Default                                                      |
+| ------------------------- | ---------------------------------------------------------- | ------------------ | -------- | ------------------------------------------------------------ |
+| activeItemBackgroundColor | Background color for the 'active' item                     | `string`           | no       | (theme.palette.type === 'light' ? primary50Color : theme.palette.primary.main) |
```

Comment:
primary50color is a variable isn't it? It would be more helpful to show the actual value I think. Or maybe for these types of props, we can just say `varies for light/dark theme`